### DISCUSSION TITLE
05/01일 멘토링 관련

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/Mentoring-Project#61] [General]
멘토님 이번 주 멘토링을 다음 주로 미룰 수 있을까요?  

정리 안된 내용도 많고 좀 더 다듬어서 멘토링 받고 싶습니다..!

직전에 연락드려서 죄송합니다..!

**Comment by jongfeel** [comment]
괜찮습니다.

그런데 제 멘토링 스케줄을 보면 아시겠지만 지금 동시에 진행하고 계신 분이 네 분이나 계시고
거기에 한이음 멘토링, academic conference 까지 껴 있어서 스케줄 잡기가 쉽지는 않을 거에요.

https://www.notion.so/jongfeel/Mentoring-schedule-9b78ed7c4289409a858a96fbea783b42

다음주라면 5월 10일 수요일에 시간이 되는데 수요일로 하겠습니다. 안되시면 얘기해 주세요.

**Reply by fkdl0048** [reply]
정말 빽빽하네요,,, 다음부턴 놓치지 않겠습니다..!  

수요일 가능해서 5월 10일 수요일 오후 11시에 뵙겠습니다..!
